runtime/tests/Test_LightningMeasures.cpp

```diff
@@ -1742,3 +1744,29 @@ TEMPLATE_LIST_TEST_CASE("Counts and PartialCounts tests with numWires=0-4 shots=
     CHECK(sum3 == shots);
     CHECK(sum4 == shots);
 }
+
+TEMPLATE_LIST_TEST_CASE("Measurement with a seeded device", "[Measures]", SimTypes)
+{
+    for (size_t _ = 0; _ < 5; _++) {
+        std::unique_ptr<TestType> sim = std::make_unique<TestType>();
+        std::unique_ptr<TestType> sim1 = std::make_unique<TestType>();
+
+        std::mt19937 gen(37);
+        sim->SetDevicePRNG(&gen);
+        std::vector<QubitIdType> Qs;
+        Qs.reserve(1);
+        Qs.push_back(sim->AllocateQubit());
+        sim->NamedOperation("Hadamard", {}, {Qs[0]}, false);
+        auto m = sim->Measure(Qs[0]);
+
+        std::mt19937 gen1(37);
+        sim1->SetDevicePRNG(&gen1);
+        std::vector<QubitIdType> Qs1;
+        Qs1.reserve(1);
+        Qs1.push_back(sim1->AllocateQubit());
+        sim1->NamedOperation("Hadamard", {}, {Qs1[0]}, false);
+        auto m1 = sim1->Measure(Qs1[0]);
+
+        CHECK(*m == *m1);
+    }
+}
```

Comment:
```suggestion
}

```

Reply:
done

Reply:
`make format` removes this new line (and CI formatter complains if it's there) (and existing tests don't have it)

Reply:
`make format` is wrong then

All our source files follow the final newline convention